tests/input_files/embeddedmesh_cantilever_beam.dat

```diff
@@ -36,6 +36,7 @@ DIVERCONT                             stop
 NAME                                  Struct_Solver
 SOLVER                                Belos
 SOLVER_XML_FILE                       xml/linear_solver/iterative_gmres_template.xml
+IFPACK_XML_FILE                       xml/preconditioner/ifpack.xml
```

Comment:
This is now a weird scenario: We specify to use Belos as `SOLVER` and rely on `AZSOLVE`'s default value GMRES. We also rely on the `AZPREC`'s default value Ifpack for the preconditioner, but have to specify the `IFPACK_XML_FILE` explicitly.

I don't have a better solution for now, but I just want to point this out. This happens in other input files as well.